client/store.js

```diff
@@ -244,6 +254,12 @@ store.setupKresus = function(cb) {
         has(world, 'accounts');
         has(world, 'operations');
 
+        let defaultAccountId = store.getDefaultAccountId();
+        if (defaultAccountId && !data.currentAccountId)
+        {
+            data.currentAccountId = defaultAccountId;
```

Comment:
IMO, it gets more readable because there's only one place where you set in bulk data.currentAccountId and data.currentBankId.

The assert will become something like the one for customfields: either the string defaultAccountId is empty, or we call the assertion.